docs/src/operate/c8y/supported-operations.md

```diff
@@ -241,9 +243,177 @@ Arguments will be parsed correctly as long as following features are not include
 
 In case those unsupported shell features are present, the syntax that introduce them is interpreted literally.
 
-Be aware that SmartREST payload is always added as the last argument. The command presented above will actually lead to following code execution
+Be aware that SmartRest payload is always added as the last argument. The command presented above will actually lead to following code execution
 
 ```bash
 python /etc/tedge/operations/command.py $SMART_REST_PAYLOAD
 ```
-:::
\ No newline at end of file
+:::
+
+#### Create operation delivered via C8Y Json
+
+There are two approaches for handling custom operation via C8Y Json.
+The first one can only be used with main device, while the second one extends it to child devices. Here is a sample of incoming C8Y Json message with custom operation fragment on topic `c8y/devicecontrol/notifications`: 
+
+```json
+{
+  "delivery": {
+    "log": [],
+    "time": "2024-07-22T10:26:31.457Z",
+    "status": "PENDING"
+  },
+  "agentId":"201802315",
+  "creationTime":"2024-09-02T19:56:01.313Z",
+  "deviceId":"201802315",
+  "id":"1800380",
+  "status":"PENDING",
+  "c8y_Command":{
+    "text":"echo helloworld"
+  },
+  "description":"Execute command",
+  "externalSource":{
+    "externalId":"test-device",
+    "type":"c8y_Serial"
+  }
+}
+```
+
+The first approach assumes that user will create the operation file with the name of the operation:
+
+```sh
+sudo -u tedge touch /etc/tedge/operations/c8y/c8y_Command
+```
+
+... with the definition containing `on_fragment` field:
+
+```toml title="file: /etc/tedge/operations/c8y/c8y_Command"
+[exec]
+topic = "c8y/devicecontrol/notifications"
+on_fragment = "c8y_Command"
+command = "/etc/tedge/operations/command ${.payload.c8y_Command.text}"
+skip_status_update = true
```

Comment:
I prefer to keep it `false` as it is the default value.

```suggestion
skip_status_update = false
```

Reply:
Resolved by a267f04581ec93b1644543df4d45e998b590aaa8